sql/parser/builtins.go

```diff
@@ -1161,6 +1186,31 @@ func regexpEvalFlags(pattern, sqlFlags string) (string, error) {
 	return fmt.Sprintf("(?%s:%s)", flagString.String(), pattern), nil
 }
 
+func overlay(s, to string, pos, size int) (Datum, error) {
+	if pos < 1 {
+		return nil, errors.New("negative substring length not allowed")
```

Comment:
Yeah, I stole that phrasing directly from postgres. I'll reword it and add in `pos`.